lib/AsyncStorage.js

```diff
@@ -353,7 +353,7 @@ const AsyncStorage = {
     callback?: ?(errors: ?$ReadOnlyArray<?Error>) => void,
   ): Promise<null> {
     return new Promise((resolve, reject) => {
-      keys.forEach(checkValidInput);
+      keys.forEach(key => checkValidInput(key, 'value'));
```

Comment:
```suggestion
      keys.forEach(key => checkValidInput(key));
```

You don't have to pass a second argument here